src/Broadcast.js

```diff
@@ -28,6 +28,26 @@ class Broadcast {
     });
   }
 
+  static atExcept(source, message, excludes = [], wrapWidth, useColor, formatter) {
```

Comment:
excludes shouldn't be optional, either they're excluding a list or they're not. And just as a general rule it's bad practice to have optional parameters between required parameters